abigen/go.mod

```diff
@@ -0,0 +1,42 @@
+module github.com/cosmos/solidity-ibc-eureka/abigen
+
+go 1.22.7
```

Comment:
Using the setup as the e2e tests. But maybe we should just fully update it to go 1.23 soon...